src/app/legacy/components/MediaPlayer/index.stories.jsx

```diff
@@ -47,8 +45,8 @@ export const MAPCanonical = () => (
         type: 'video',
         ...withDuration,
       }}
-      showLoadingImage={boolean('Show loading image', true)}
-      darkPlaceholder={boolean('Dark placeholder', false)}
+      showLoadingImage
+      darkPlaceholder
```

Comment:
Should these be controls now?

Reply:
I don't think they should have been controls in the first place as the MAP version of the media player should always render in the dark UI.

Reply:
Makes sense, thanks!